docs/lib/analytics/fragments/ios/record.md

```diff
@@ -41,35 +40,36 @@ Events have default configuration to flush out to the network every 60 seconds.
 }
 ```
 
-> **Note**: If you set `autoFlushEventsInterval` to 0, you are responsible for calling `Amplify..flushEvents()` to flush events.
+> **Note**: If you set `autoFlushEventsInterval` to 0, you are responsible for calling `Amplify.Analytics.flushEvents()` to submit the recorded events to the backend.
 
 
 ## Global Properties
 
-You can register properties which will be used across all `Amplify.Analytics.record`.
+You can register properties which will be used across all `Amplify.Analytics.record(event:)` calls.
 
 ```swift
-let globalProperties = ["globalPropertyKey": "value"] as [String: AnalyticsPropertyValue]
+let globalProperties: [String: AnalyticsPropertyValue] = ["globalPropertyKey": "value"]
 Amplify.Analytics.registerGlobalProperties(globalProperties)
 ```
 
-To unregister all global properties call `Amplify.Analytics.unregisterGlobalProperties()`. 
+To unregister all global properties call `Amplify.Analytics.unregisterGlobalProperties()`:
 
-To unregister a single property:
 ```swift
 Amplify.Analytics.unregisterGlobalProperties(["globalPropertyKey"])
 ```
 
 ## Disable Analytics
 
 To disable analytics:
+
 ```swift
 Amplify.Analytics.disable()
 ```
 
 ## Enable Analytics
 
 To enable analytics:
```

Comment:
Nit: you may want to say re-enable to not imply a user must call this to initialize analytics.